Automattic/jetpack
# Ads: Summer Ads Updates

Porting omnibus WordAds updates to Jetpack. Notably this update includes:

- **Cleanup & Optimizations**
- **2nd Belowpost Ad Option**
<img width="426" alt="screen shot 2017-08-23 at 1 04 06 pm" src="https://user-images.githubusercontent.com/273708/29635946-a53a56d2-8803-11e7-88de-d1d3d98260d7.png">
<img width="703" alt="screen shot 2017-08-23 at 1 04 22 pm" src="https://user-images.githubusercontent.com/273708/29635950-a8d8a654-8803-11e7-8957-2c954a9f2bf6.png">

- **Criteo Adblock Acceptable Ads fallback**
<img width="1307" alt="screen shot 2017-08-23 at 1 06 30 pm" src="https://user-images.githubusercontent.com/273708/29636023-efc3b8ec-8803-11e7-9668-495460e244d4.png">

- **Granular Display Controls**
<img width="722" alt="screen shot 2017-08-23 at 1 07 43 pm" src="https://user-images.githubusercontent.com/273708/29636079-2ed445ba-8804-11e7-9e34-56c38bf8a335.png">

**To Test**
1. Enable Jetpack Ads module
1. Enable `Display second ad below post`
    1. Check to see that 2nd unit appears below post
1. Enable options under `Enable below post ads on`
    1. Check below post ads do/don't display under appropriate type of page
1. Enable AdBlock Plus extension (😭)
    1. Check `Allow some non-intrusive advertising` in `Adblock Plus Options`
    1. Check that some non-obtrusive ads are in the regular spots.
<img width="607" alt="screen shot 2017-08-23 at 1 18 40 pm" src="https://user-images.githubusercontent.com/273708/29636515-bbb2064c-8805-11e7-8a2a-b798a28d7b4b.png">
